parser.py

```diff
@@ -109,12 +112,21 @@ def match_subject(tag):
     def course_action(self, course_unique):
         """Return the action for the course unique"""
         tag = self.soup.find("a", id=self.ALL_COURSES, text=course_unique)
+
         if not tag:
             logging.warning(u"Couldn't find the course '{0}'".format(course_unique))
             return None
 
         return tag["id"]
 
+    def disambiguation_action(self):
+        tags = self.soup.find_all("a", id=self.ALL_CAREERS)
+
+        if not tags:
+            return None
+
+        return tags[len(tags)-1]["id"]
```

Comment:
It just feels like it might be nicer to not make this be hard-coded, and make the detection more flexible.  Not super important for this version though.  I imagine they are alphabetical, and undergraduate is the last in the alphabet.